src/views/Team/Outlet.vue

```diff
@@ -101,235 +193,292 @@
 </template>
 
 <script lang="ts">
-import RenderMd from '@/components/render-md.vue';
-import Avatar from '@/components/avatar.vue';
-import { DefaultIcon, Icons } from '@/components/generic/inline-link-list.vue';
-import InlineLinkList from '@/components/generic/inline-link-list.vue';
-import EditLinks from '@/components/settings/edit-links.vue';
-import CreateSubTeam from '@/components/settings/create-subteam.vue';
-import GenericEditorModal from '@/components/settings/generic-editor-modal.vue';
-import Activity from '@/components/activity.vue';
-import Qrcode from '@/components/qrcode.vue';
+// import RenderMd from "@/components/render-md.vue";
+import Avatar from "@/components/avatar.vue";
+import { DefaultIcon, Icons } from "@/components/generic/inline-link-list.vue";
+import InlineLinkList from "@/components/generic/inline-link-list.vue";
+import EditLinks from "@/components/settings/edit-links.vue";
+import CreateSubTeam from "@/components/settings/create-subteam.vue";
+import GenericEditorModal from "@/components/settings/generic-editor-modal.vue";
+import Activity from "@/components/activity.vue";
+import Qrcode from "@/components/qrcode.vue";
 import {
-  IonContent, IonPage,  IonIcon, IonChip, IonSpinner, IonButton, IonToolbar,
-  modalController, alertController,
-} from '@ionic/vue';
+  IonContent,
+  IonPage,
+  IonIcon,
+  IonChip,
+  IonSpinner,
+  IonButton,
+  IonToolbar,
+  modalController,
+  alertController,
+  IonSegmentButton,
+  IonSegment,
+  IonLabel,
+} from "@ionic/vue";
+
 import {
-  chatbubbles, logoWhatsapp, cloudUploadOutline,
+  chatbubbles,
+  logoWhatsapp,
+  cloudUploadOutline,
   trashOutline as trashIcon,
   imageOutline as imageIcon,
   qrCodeOutline as qrCodeIcon,
   createOutline as editIcon,
   addCircleOutline as addIcon,
-} from 'ionicons/icons';
-import { defineComponent } from 'vue';
-import { useStore } from '@/stores/';
-import Parse from 'parse';
-import { takePicture, Photo } from '@/utils/camera';
-import { Model } from '@/utils/model';
-import { absoluteUrl } from '@/utils/url';
+} from "ionicons/icons";
+import { defineComponent } from "vue";
+import { useStore } from "@/stores/";
+import Parse from "parse";
+import { takePicture, Photo } from "@/utils/camera";
+import { Model } from "@/utils/model";
+import { absoluteUrl } from "@/utils/url";
 
 const DEFAULT_STYLES = {
-  "background": "transparent",
-  "backgroundImage": "linear-gradient(to right, var(--ion-color-secondary) 0%, var(--ion-color-primary) 100%)"
-}
+  background: "transparent",
+  backgroundImage:
+    "linear-gradient(to right, var(--ion-color-secondary) 0%, var(--ion-color-primary) 100%)",
+};
 
 export default defineComponent({
-  name: 'Team',
-  data(){
+  name: "Team",
+  data() {
     return {
-      showQr: false,
       loading: true,
-    }
+      state: "about",
+      subteam: "",
```

Comment:
what's this? I think this isn't supposed to be here (but if anything, should be a getter).